### PR TITLE
Productlisting page

### PR DESCRIPTION
**Pull request by SadiqZak**
Added Routing, DB, Card components

### CODE REVIEW

File: src/Backend/db/data.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
+const data=[
+    {
+        id:"1",
+        category:"shoes",
+        title:"Birk",
+        img: require('../../Assets/Shoes/birk.jpg'),
```
Comment (Devansu-Yadav) [inline src/Backend/db/data.js:6]: Please avoid using `require()` to fetch the image. Use the source URL for the image instead.

File: src/Pages/ProductListingpage/ProductListingpage.jsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import Sidebar from "./Sidebar/Sidebar";
+import data from "../../Backend/db/data";
+import Badge from "../../Components/Header/Badge/Badge";
+
+const ProductListingpage = () => {
+  return (
+    <div className="productlist-container">
+      <Sidebar />
+      <div className="recommended-videos">
```
Comment (Devansu-Yadav) [inline src/Pages/ProductListingpage/ProductListingpage.jsx:10]: Please rename the CSS class accordingly.

**Review by Devansu-Yadav — APPROVED** [review]
Like:
- Good color combination used for the UI
- Good to see that you have broken the UI down into separate elements. Will help to easily debug code.

Suggestions:
- Please review the comments that I have made above.
- Product listing page needs to be made responsive for mobile screens.
- Avoid using `require()` to fetch images. I'd suggest using Image URLs directly instead.

**Review by saketkum11 — APPROVED** [review]
-  clean code
-  much understandable.
-  UI looks Great.
-  Color choice is good.

**Pull request merged**